src/adapters/supabase/helpers/client.ts

```diff
@@ -152,7 +152,7 @@ export const upsertUser = async (user: UserProfile): Promise<void> => {
 /**
  * Performs an UPSERT on the wallet table.
  * @param username The user name you want to upsert a wallet address for
- * @param address The EVM wallet address
+ * @param address The account address
  */
 export const upsertWalletAddress = async (username: string, address: string): Promise<void> => {
```

Comment:
One of my colleagues had a security related concern about updating wallet addresses. Can you walk me through how an exploit could occur, and what measures are being put into place to combat these vectors?

The scenario he presented was if somebody is able to update all the wallets in our system and then front-run/claim new bounties posted. I think its unlikely to occur but I didn't have an answer for him as I am not 100% clear on the full architecture related to this system.

Reply:
we can keep talking on the discussion